frontend/src/host_orchestrator/orchestrator/controller.go

```diff
@@ -177,7 +182,11 @@ func (h *createCVDHandler) Handle(r *http.Request) (interface{}, error) {
 	artifactsFetcher := newBuildAPIArtifactsFetcher(buildAPI)
 	cvdBundleFetcher := newFetchCVDCommandArtifactsFetcher(exec.CommandContext, creds)
 	cvdStartTimeout := 3 * time.Minute
-	if req.EnvConfig != nil {
+	if isRunningInDocker() {
+		// Use a lengthier timeout when running within a docker because it could race resource with
+		// other containers and take much longer time than normal status.
+		cvdStartTimeout = 30 * time.Minute
```

Comment:
Is this going to be a valid timeout every time it's run in docker? Or is it more relevant to the setup you're using?
I think this should be a configuration property in `/etc/defaults/cuttlefish-host-orchestrator`. Also, 30 minutes seems excessive in comparison to the 7 minutes below.

Reply:
I ran CVD with docker at the setup I'm using and it took 15 mins to show up VIRTUAL_DEVICE_BOOT_COMPLETED because of using many resources.

I changed the code to use default configuration file to make it configurable.